app/controllers/users_controller.rb

```diff
@@ -1,12 +1,15 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
+  before_action :correct_user, only: [:edit, :update]
```

Comment:
nên dùng %i thay cho array symbol nha
```suggestion
  before_action :correct_user, only: %i(edit update)
```